op-conductor/client/rollupboost.go

```diff
@@ -0,0 +1,69 @@
+package client
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/ethereum/go-ethereum/log"
+	"github.com/ethereum/go-ethereum/rpc"
+)
+
+type ExecutionMode bool
+
+type RollupBoostControl interface {
+	// GetExecutionMode gets the current execution mode of rollup boost
+	GetExecutionMode(ctx context.Context) (bool, error)
+	// SetExecutionMode sets the execution mode of rollup boost
+	SetExecutionMode(ctx context.Context, enabled bool) error
+}
+
+// RollupBoostControlClient implements RollupBoostControl
+type RollupBoostControlClient struct {
+	client *rpc.Client
+	log    log.Logger
+}
+
+var _ RollupBoostControl = (*RollupBoostControlClient)(nil)
+
+// NewRollupBoostControlClient creates a new RollupBoostControlClient
+func NewRollupBoostControlClient(ctx context.Context, url string, log log.Logger) (*RollupBoostControlClient, error) {
+	httpClient := &http.Client{
+		Timeout: 10 * time.Second,
+	}
```

Comment:
this is not needed? As timeout was provided when calling it?

Reply:
or better, make it consistent across different calls, I saw there's 5s / 10s timeout in different places